apiserver/export_test.go

```diff
@@ -26,7 +26,6 @@ var (
 	BZMimeType            = bzMimeType
 	JSMimeType            = jsMimeType
 	GUIURLPathPrefix      = guiURLPathPrefix
-	SpritePath            = spritePath
```

Comment:
The spritePath was removed in the main source file, so needed to be removed from the tests as well